src/themes/default/elements/step.variables

```diff
@@ -101,6 +117,30 @@
 @orderedFontFamily: inherit;
 @orderedFontWeight: @bold;
 
+/* Circular */
+@circularStepsMargin: @stepMargin;
+@circularStepsPadding: @stepMargin;
+@circularStepLineMargin: 2.2em;
+@circularStepDisabledItemOpacity: 0.5;
+@circularStepColor: @borderColor;
+@circularStepBorderWidth: 2px;
+@circularStepBorder: @circularStepBorderWidth solid @circularStepColor;
+@circularStepRingSize: 2.2em;
+@circularStepIconSize: 0.95em;
+@circularStepIconDistance: calc((-@circularStepRingSize + @circularStepIconSize) - 0.35em);
```

Comment:
reducing the distance to 0.3 instead of 0.35 made the inner circle look better centered to me (especially when compared to vertical variant)

0.35em:
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/da7aa8aa-2546-48f4-abf5-70fa4a22fc57)

0.3em:
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/5525252e-097b-4491-a99d-108cdaf29c79)

vertical (untouched)
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/b0e47955-ad8a-4f14-80b4-f14776bc96e9)